src/propagate-job.ts

```diff
@@ -77,7 +80,7 @@ export async function run(): Promise<void> {
       //       this will return a priority fee of 10 GWEI. We need to pass it so that it properly calculated the maxFeePerGas
       const { priorityFeeInGwei, maxFeePerGas } = getMainnetGasType2Parameters({
         block,
-        blocksAhead,
+        blocksAhead: 0,
```

Comment:
i'd suggest calculating `blocksAhead` as: `FUTURE_BLOCKS + BURST_SIZE` and setting `FUTURE_BLOCKS` to at least `2`. 

bundles take time to propagate so they never get included in the same block they're sent. 

by setting to `blocksAhead` to say `5` here, the gas parameters will be slightly higher than they should, but they give us room for error in case the incoming blocks are not from flashbots.

Reply:
I was confused about this because we aren't sending any type of multi-tx bundles. Does it matter?